packages/react-compose-next/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@fluentui/react-compose-next",
+  "version": "7.0.0",
```

Suggest v0 until we're ready

```suggestion
  "version": "0.1.0",
```